Makefile

```
@@ -33,3 +33,8 @@ test:
 .PHONY: boilerplate-update
 boilerplate-update:
 	@boilerplate/update
+
+.PHONY: golden-ami-%
+golden-ami-%:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)